src/pubSubClient.js

```diff
@@ -16,20 +16,36 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-const logger = require('./utils/Logger')('pubSubClient.js');
+const { v4: uuidv4 } = require('uuid');
+const logger = require('../src/utils/Logger')('pubSubClient.js');
+require('dotenv').config({ override: true });
 
 class PubSubClient {
   constructor() {
     this.ws = null;
     this.url = 'ws://your-ws-url-here.com';
+    this.PUBSUB_SUBSCRIBE_TOPIC_SUFFIX = '_FCS';
+    this.PUBSUB_PUBLISH_TOPIC_SUFFIX = '_FCA';
   }
 
   // Initializes a WS connection
   async initialize() {
+    const appUrl = window.location;
+    let pubSubTopicUUID = new URLSearchParams(appUrl.search).get('pubsub_uuid');
+
+    // If pubsub_uuid isn't passed as a query param, use a default value
+    if (!pubSubTopicUUID) {
+      pubSubTopicUUID = 'DEFAULT_TOPIC';
```

Comment:
Don't we get the device MAC and use that as the default?

Reply:
PubSubTopicUUID should have first priority if it is passed in. Then I added logic to use the deviceMac